### PR TITLE
Feat: Initial population of the customer controller

### PR DESCRIPTION
**Pull request by ayaseef**
I just added the index controller action for customer controller. I also deleted the videos yml file

### CODE REVIEW

File: app/controllers/customers_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class CustomersController < ApplicationController
+
```
Comment (beauttie) [inline app/controllers/customers_controller.rb:2]: will remove white space

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  #
```
Comment (mmcknett) [inline config/routes.rb:3]:
```suggestion
```

**Review by mmcknett — COMMENTED** [review]
Just a nit pick

File: db/seeds.rb [hunk 3]
```diff
@@ -5,8 +5,8 @@
...
-  Video.create!(video)
-end
-
-puts "The database now has #{Video.count} videos"
\ No newline at end of file
+# JSON.parse(File.read('db/seeds/videos.json')).each do |video|
+#   Video.create!(video)
+# end
+#
+# puts "The database now has #{Video.count} videos"
```
Comment (mmcknett) [inline db/seeds.rb:12]:
```suggestion
```

**Review by beauttie — APPROVED** [review]
great job

**Pull request merged**